api/src/main/java/module-info.java

```diff
@@ -237,7 +237,7 @@
  * <br>{@code int} and {@link Integer}
  * <br>{@code long} and {@link Long}
  * <br>{@code short} and {@link Short}</td>
- * <td></td></tr>
+ * <td>Boolean values are sorted such that {@code false} &lt; {@code true}.</td></tr>
```

Comment:
Do we have this control?

E.g: I do believe that SQLite, DynamoDB and CouchDB & JSON-based Databases.

But I did go deep on this topic.

Reply:
@otaviojava I am not adding any new requirement here.  This same requirement already exists in the specification, section [5.4.6 Ordering](https://jakarta.ee/specifications/data/1.0/jakarta-data-1.0#_ordering) where it states,
> Boolean values must be ordered so that `false < true` is satisfied.

This PR only copies the existing requirement to the Javadoc so that users can more easily find the information.

Reply:
Ok, but at NoSQL, there is no this guarantee. Unless it queries by memory, it is not the most efficient way to do it.

Anyway, once, those are exceptions. I guess it it fine.